stl/inc/xutility

```diff
@@ -184,6 +184,15 @@ struct _Get_rebind_alias<_Ty, _Other, void_t<typename _Ty::template rebind<_Othe
     using type = typename _Ty::template rebind<_Other>;
 };
 
+#if _HAS_CXX23
+template <class _Raw, class _Source>
+_NODISCARD constexpr auto&& _Forward_as(auto&& _Val) noexcept {

```

Comment:
Should this be
```suggestion
_NODISCARD constexpr auto&& _Forward_as(_Source&& _Val) noexcept {
```

Reply:
I guess not. If this takes `_Source&&`, then the caller will need to use `std::forward`, which is not a clear win.

Reply:
Yeah, maybe it is just the mixture of abbreviated template and old style template syntax that is throwing me off

Reply:
I wanted this to accept both lvalues and rvalues, but without having two overloads like `std::forward`. I also didn't want to deduce the source type so as to force callers to pass both source and target types explicitly via template argument list. This function template is doing something fairly sketchy, I don't want any implicit behaviors that make it easier to call without thinking very carefully first.